content/docs/next-release/operator.md

```diff
@@ -389,6 +389,7 @@ storage:
     numberOfDays: 7                               // number of days to wait before deleting a record
     schedule: "55 23 * * *"                       // cron expression for it to run
     image: jaegertracing/jaeger-es-index-cleaner  // image of the job
+    successfulJobsHistoryLimit: 0                 // number of successful job will will be kept in history, support for esRollover, esIndexCleaner, dependencies.
```

Comment:
Could you please add this also to the last releases if the feature was available there too?

Reply:
after merge https://github.com/jaegertracing/jaeger-operator/pull/621, I will add it to the latest release. Thanks

Reply:
Hi, https://github.com/jaegertracing/jaeger-operator/pull/621 was merged yesterday

Reply:
The comment does not seem right:  // number of successful job will will be kept in history,` It should rather be ` // number of successful jobs that will be kept in history,`